osrf_msgs/JointCommands joint_commands
---
bool success
string status_message
